include/quicr/detail/messages.h

```diff
@@ -62,8 +62,7 @@ namespace quicr::messages {
         CLIENT_SETUP = 0x40,
         SERVER_SETUP,
 
-        STREAM_HEADER_TRACK = 0x50,
-        STREAM_HEADER_GROUP,
+        STREAM_HEADER_GROUP = 0x51,
```

Comment:
Considering PR #289, per group goes away as well as per track. It's fine to have separate PRs, but merging needs to be done in order.